plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/EditMavenArtifactDialog.java

```diff
@@ -128,18 +222,152 @@ private void updateButtons() {
         );
     }
 
+    private List<DriverLibraryMavenArtifact> parseGradle() throws XMLException {
+        final List<DriverLibraryMavenArtifact> artifacts = new ArrayList<>();
+        final Matcher matcher = REGEX_FOR_GRADLE.matcher(fieldText.getText());
+        while (matcher.find()) {
+            final String classifier = matcher.group("group");
+
+            // Try to match long form
+            String group = matcher.group("group");
+            String name = matcher.group("name");
+            String version = matcher.group("version");
+
+            if (!CommonUtils.isNotEmpty(group) && !CommonUtils.isNotEmpty(name) && !CommonUtils.isNotEmpty(version)) {
+                // Try to match short form instead
+                group = matcher.group("shortGroup");
+                name = matcher.group("shortName");
+                version = matcher.group("shortVersion");
+            }
+
+            if (CommonUtils.isNotEmpty(group) && CommonUtils.isNotEmpty(name) && CommonUtils.isNotEmpty(version)) {
+                DriverLibraryMavenArtifact lib = new DriverLibraryMavenArtifact(EditMavenArtifactDialog.this.driver, DBPDriverLibrary.FileType.jar, "", "");
+                lib.setReference(new MavenArtifactReference(CommonUtils.notEmpty(group), CommonUtils.notEmpty(name), null, CommonUtils.notEmpty(version)));
+                artifacts.add(lib);
+            } else {
+                throw new XMLException("Wrong parameters!");
```

Comment:
I'm pretty confident that we don't parse XML here. Moreover, I believe that the text of the exception should more clearly indicate what's wrong.